daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md

```diff
@@ -90,6 +89,13 @@ The above example uses secrets as plain strings. It is recommended to use a [sec
 | processMode | N | Enable processing multiple messages at once. Default: `"async"` | `"async"`, `"sync"`|
 | subscribeType | N | Pulsar supports four kinds of [subscription types](https://pulsar.apache.org/docs/3.0.x/concepts-messaging/#subscription-types). Default: `"shared"` | `"shared"`, `"exclusive"`, `"failover"`, `"key_shared"`|
 | partitionKey | N | Sets the key of the message for routing policy. Default: `""` | |
+| token | N  | Token used for authentication. | [How to create Pulsar token](https://pulsar.apache.org/docs/en/security-jwt/#generate-tokens)|
```

Comment:
Consider creating a separate table for each authentication profile? (and include a mention that it can work without any). Example: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-sqlserver/#spec-metadata-fields

Reply:
@JoshVanL - Can you address this comment from @ItalyPaleAle?

Reply:
@JoshVanL - ping

Reply:
@msfussell all comments look to be resolved from my side I think.